program/src/processor.rs

```diff
@@ -95,7 +98,13 @@ pub fn process_create_mint(
         wrapped_token_program_account.key,
         &bump_seed,
     );
-    let space = spl_token_2022::state::Mint::get_packed_len();
+
+    let extensions = if *wrapped_token_program_account.key == spl_token_2022::id() {
+        mint_customizer.get_extension_types()
+    } else {
+        vec![]
+    };
+    let space = ExtensionType::try_calculate_account_len::<Mint>(&extensions)?;
```

Comment:
This works for now, but it'll need some extra logic for metadata in the future, since metadata has a variable length

Reply:
Ah! Instead of requesting a list of extensions, I'll swap the interface to return space instead 👍